source/whats-new.txt

```diff
@@ -42,6 +51,25 @@ The {+driver-short+} v4.11 release includes the following new features:
 - Adds support for free-threaded CPython when running Python v3.13+. 
   For more information about free threading, see the 
   `Python documentation <https://docs.python.org/3/howto/free-threading-python.html>`__.
+- In-use encryption requires ``pymongocrypt`` v1.12 or later.
+- ``MongoClient.address`` and ``AsyncMongoClient.address`` now correctly block
```

Comment:
assuming these are methods?

```suggestion
- The ``MongoClient.address()`` and ``AsyncMongoClient.address()`` methods correctly block
```

Reply:
I've been spending too long in Mongoid land apparently